### PR TITLE
Delete composer.lock

### LINKED ISSUE #1175: Remove composer.lock from profile

### PR DESCRIPTION
**Pull request by Saphyel**
Fixes #1175

## Changes proposed in this PR

- Briefly describe the changes in this PR
- Do you have some test asserting this behavior?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,9 @@
             "@grunt:dependencies"
         ],
         "post-update-cmd": [
-            "@drupal:scaffold"
+            "@drupal:scaffold",
+            "@campaign:scaffold",
+            "@grunt:dependencies"
```
Comment (pvhee) [inline composer.json]: @Saphyel why do we need to change this in this PR? Seems unrelated?

Comment (Saphyel) [reply composer.json]: it's related because now composer executes `update` (even if we run `composer install` see: 91d891b )
